generator/src/test/java/com/scottlogic/deg/generator/commandLine/CommandLineValidatorTest.java

```diff
@@ -0,0 +1,77 @@
+package com.scottlogic.deg.generator.commandLine;
+
+import com.scottlogic.deg.generator.CommandLine.CommandLineValidator;
+import com.scottlogic.deg.generator.CommandLine.ValidationResult;
+import com.scottlogic.deg.generator.generation.GenerationConfig;
+import com.scottlogic.deg.generator.generation.TestGenerationConfigSource;
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+public class CommandLineValidatorTest {
+    @Test
+    public void validConfigShouldReturnValid() {
+        //Arrange
+        GenerationConfig config = new GenerationConfig(
+            new TestGenerationConfigSource(
+                GenerationConfig.DataGenerationType.INTERESTING,
+                GenerationConfig.TreeWalkerType.REDUCTIVE,
+                GenerationConfig.CombinationStrategyType.PINNING
+                )
+        );
+        CommandLineValidator validator = new CommandLineValidator();
+        ArrayList<String> errorsReturned = new ArrayList<String>();
+
+        //Act
+        ValidationResult validationResult = validator.validateCommandLine(config);
+
+        //Assert
+        Assert.assertTrue(validationResult.isValid());
+        Assert.assertEquals(validationResult.errorMessages, errorsReturned);
```

Comment:
Would be better expressed as `Assert.assertIsEmpty()`

Reply:
`Assert.assertTrue(validationResult.errorMessages.isEmpty());` is this better? I can't see an `assertIsEmpty` option

Reply:
See https://www.mkyong.com/unittest/junit-how-to-test-a-list/
where it describes:

`Assert.assertThat(validationResult.errorMessages, is(empty()));`
or equally correct
`Assert.assertThat(validationResult.errorMessages, empty());`